src/main/java/seedu/address/logic/parser/AddPersonCommandParser.java

```diff
@@ -60,9 +56,12 @@ public AddPersonCommand parse(String args) throws ParseException {
             characteristics = ParserUtil.parseCharacteristics(argMultimap.getValue(PREFIX_CHARACTERISTICS).get());
         }
 
-        Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+        Priority priority = new Priority("normal");
+        if (argMultimap.getValue(PREFIX_TAG).isPresent()) {
```

Comment:
Maybe it would be clearer if we simply don't use "Tag" at all since we now call it `Priority`. We could call it `PREFIX_PRIORITY`and change the CLI syntax to `-priority`?